src/components/Menu.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+import '../index.css';
+import './MatrixText'
+import { MatrixText } from "./MatrixText";
+
+export const Menu = (props) => {
+  return (
+    <div>
+      <div class={"menu_sidebar"}>
+        {/* This is the sidebar div  */}
+        <ul>
+        <li><Link to="/"><MatrixText text="home " length={5} speed={7} ></MatrixText></Link></li>
+          <li><Link to="/about"><MatrixText text="about " length={6} speed={7} ></MatrixText></Link></li>
+          <li><Link to="/music"><MatrixText text="music " length={6} speed={6}></MatrixText></Link></li>
+          <li><Link to="/photos"><MatrixText text="photos " length={7} speed={5}></MatrixText></Link></li>
+          <li><Link to="/shows"><MatrixText text="shows " length={6} speed={4}></MatrixText></Link></li>
+          <li><Link to="/videos"><MatrixText text="videos " length={7} speed={3}></MatrixText></Link></li>
+          <li><Link to="/contact"><MatrixText text="contact " length={8} speed={2}></MatrixText></Link></li>
```

Comment:
i'd make the speed lower? it kinda just immediately glitches in, like 1 might be good for all of them